### ISSUE TITLE
Use scientific notation for fmt::Display and fmt::Debug please

### ISSUE THREAD

**Issue by LambdaAlpha** [akubera/bigdecimal-rs#91]
Currently, BigDecimal will generate huge string if the exponent is large.
```
println!("{}", BigDecimal::from_str_radix("1.2e9999999999", 10).unwrap());
```

**Comment by mschlaipfer** [comment]
I think this is a critical bug. Printing a `BigDecimal` causes exponential blowup. It's very easy to get this wrong.

**Comment by declanvk** [comment]
@akubera is this resolved by the changes release in `0.4.3`, namely #121 ?

**Comment by akubera** [comment]
Yes.
